modular_zzplurt/code/modules/client/click.dm

```diff
@@ -0,0 +1,20 @@
+/mob/living/carbon/human
+	var/_last_next_move = 0
+
+/mob/living/carbon/human/ClickOn(atom/A, params)
+	if(world.time <= next_click)
+		return
+	_last_next_move = next_move
```

Comment:
Could you leave a comment explaining what this does and why it's necessary? Looks like it's used to roll back `next_move` later, but why's that necessary? (I'm sure it is, I just want it documented for maintainability reasons.)